infra/main.bicep

```diff
@@ -47,9 +47,11 @@ param formRecognizerSkuName string = 'S0'
 param gptDeploymentName string // Set in main.parameters.json
 param gptDeploymentCapacity int = 30
 param gptModelName string = 'gpt-35-turbo'
+param gptModelVersion string = '0613'
```

Comment:
I think @jongio wanted this, keeping the version numbers as parameters next to the model names, since they're so related. Also, I had to update the version from 0310 to 0613, as it wouldn't let me deploy new 0301s. I'm still trying to figure out differences between the two, i.e. if 0613 is always better.

Reply:
@pablocastro Was there a reason to use 0310 over 0613 for our use case?